Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -869,6 +868,10 @@ public class TimeSlider: UIControl {
             scene = sceneView.scene
         }
         
+        // Set observeGeoView after setting map and scene
+        // setting map and scene
```

Comment:
It looks like this line is a partial copy of the previous line.

Reply:
fixed